security-analysis/security-analysis-api/src/main/java/com/powsybl/security/action/HvdcAction.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.security.action;
+
+import com.powsybl.iidm.network.HvdcLine;
+
+import java.util.Optional;
+import java.util.OptionalDouble;
+
+/**
+ * an action to modify hvdc parameters
+ * parameter enabled corresponds to the regulation mode true correspond to ac emulation, false to fix target
+ * droop and p0 are parameters used for ac emulation only.
+ * activePowerSetpoint is for fix target mode only.
+ * ac emulation works only with vsc technology.
+ * attribute relative value if true add target p to the previous target p if false it replaces it
+ *
+ * @author Etienne Lesot <etienne.lesot@rte-france.com>
+ */
+public class HvdcAction extends AbstractAction {
+
+    public static final String NAME = "HVDC";
+
+    private final String hvdcId;
+    private final boolean acEmulationEnabled;
+    private final Double activePowerSetpoint;
+    private final HvdcLine.ConvertersMode converterMode;
+    private final Double droop;
+    private final Double p0;
+    private final Boolean relativeValue;
+
+    public HvdcAction(String id, String hvdcId, Double activePowerSetpoint, HvdcLine.ConvertersMode converterMode, Boolean relativeValue) {
```

Comment:
I think here you need also the enalbled boolean to deactivate the HVDC AC emulation.